autofit/non_linear/paths/abstract.py

```diff
@@ -414,14 +414,16 @@ def _save_search(self, config_dict):
             json.dump(config_dict, f, indent=4)
 
     def save_summary(self, samples, log_likelihood_function_time):
-        text_util.results_to_file(
+
+        result_info = text_util.result_info_from(
             samples=samples,
-            filename=path.join(
-                self.output_path,
-                "model.results"
-            )
         )
 
+        filename = path.join(self.output_path, "model.results")
```

Comment:
We should start using pathlib rather than joining paths

Reply:
Yeah I need to learn pathlib.